src/panels/lovelace/entity-rows/hui-cover-entity-row.ts

```diff
@@ -0,0 +1,76 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+
+import "../components/hui-generic-entity-row.js";
+import "../../../components/ha-cover-controls.js";
+import "../../../components/ha-cover-tilt-controls.js";
+
+import { isTiltOnly } from "../../../util/cover-model.js";
+import { HomeAssistant } from "../../../types.js";
+import { EntityRow, EntityConfig } from "./types.js";
+
+class HuiCoverEntityRow extends LitElement implements EntityRow {

```

Comment:
add `export` before class. We want people to be able to import this class in case they get the element from the DOM.